src/components/home/navbar/DropdownButton.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react'
+
+import DropdownMenu from './DropdownMenu'
+
+import '../../../styles/home/navbar/dropdown_button.css'
+
+class DropdownButton extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      isOpen: false
+    }
+    this.toggleMenu = this.toggleMenu.bind(this)
+    this.onClose = this.onClose.bind(this)
+  }
+
+  toggleMenu (event) {
```

Comment:
Just a thought but maybe we should have a discussion on making a ''toggle component" for the project. Several teams are repeating the same code...don't know if that is an issue that can be worked out this week....but just a thought.